std/meta.d

```diff
@@ -758,7 +762,7 @@ template MostDerived(T, TList...)
 }
 
 /**
- * Returns the `AliasSeq` TList with the types sorted so that the most
+ * Returns an `AliasSeq` with the elements of TList sorted so that the most
```

Comment:
Since you're modifying this line anyway, might as well change

    `AliasSeq`

to

    $(LREF AliasSeq)

Same throughout.